metadata_service/proxy/atlas_proxy.py

```diff
@@ -378,7 +378,7 @@ def get_table(self, *, table_uri: str) -> Table:
                 description=attrs.get('description') or attrs.get('comment'),
                 owners=[User(email=attrs.get('owner'))],
                 columns=columns,
-                last_updated_timestamp=table_details.get('updateTime'))
+                last_updated_timestamp=table_details.get('updateTime')/1000)
```

Comment:
@mgorsk1 can we please use python's datetime objects and functions to format timestamp ?

Reply:
not sure if that's required ? the type of last_updated_timestamp is int. made another go with formatting to unix epoch which would be less error prone.